packages/edit-site/src/components/create-template-part-modal/index.js

```diff
@@ -54,76 +55,84 @@ export default function CreateTemplatePartModal( { closeModal, onCreate } ) {
 					await onCreate( { title, area } );
 				} }
 			>
-				<TextControl
-					label={ __( 'Name' ) }
-					value={ title }
-					onChange={ setTitle }
-					required
-				/>
-				<BaseControl
-					label={ __( 'Area' ) }
-					id={ `edit-site-create-template-part-modal__area-selection-${ instanceId }` }
-					className="edit-site-create-template-part-modal__area-base-control"
-				>
-					<RadioGroup
+				<VStack spacing="4">
```

Comment:
I personally think that the new spacing looks more centered, but I could be wrong. I can add CSS to make it like trunk if others prefer.

Reply:
Looks good to me 👍